pontoon/base/utils.py

```diff
@@ -381,7 +381,14 @@ def get_download_content(slug, code, part):
         dirnames = set([locale.code, locale.code.replace('-', '_')])
         locale_path = _download_file(locale_prefixes, dirnames, resource.path)
         if not locale_path:
```

Comment:
I would add `and not resource.is_asymmetric` here and an whole code related to is_assymetric  into this block: https://github.com/mozilla/pontoon/pull/360/files#diff-cfad39fede62fdc24a97edc6622b538dL388 I think this change would make reading of this code a little simpler.